### PR TITLE
AntodeLicensingPageChanges

### PR DESCRIPTION
**Pull request by Antoine2F**
modify response to #### What happens if a user turns on/off AI Builder per user capacity add-on license? add question/answer #### What happens to my data and models when my AI Builder trial expires? modify response to #### Can I block users in my organization from signing up for an AI Builder trial?

### CODE REVIEW

File: ai-builder-docs/administer-licensing.md [hunk 1]
```diff
@@ -168,15 +168,19 @@ To learn more, go to [About trial environments](/power-platform/admin/trial-envi
 
 #### Can I block users in my organization from signing up for an AI Builder trial?
 
-Any individual can try out the features of AI Builder for 30 days and incur no costs to your company. This option is available to any user in a tenant and can't be disabled by an admin.
+Any individual can try out the features of AI Builder for 30 days and incur no costs to your company. This option is available to any user in a tenant and can't be disabled by an admin. Note that once some AI credits is purchased by your company (either purchasing Add On or included in some licenses), then trial are not proposed to users any more.
```
Comment (chplanty) [inline ai-builder-docs/administer-licensing.md:171]: "anymore" or "any more" ?

File: ai-builder-docs/administer-licensing.md [hunk 2]
```diff
@@ -168,15 +168,19 @@ To learn more, go to [About trial environments](/power-platform/admin/trial-envi
...
 Each user license grants you 5,000 credits, allowing you to assess the capabilities in AI Builder. For instance, you could use these credits to extract data from a few documents with *document processing* or perform hundreds of basic OCR extractions with *text recognition*.
 
 #### What happens if a user turns on/off AI Builder per user capacity add-on license?
 
-The Power Apps per user license includes 500 seeded AI Builder credits. If **AI Builder capacity Per User add-on** isn't selected, the 500 AI Builder credits won't be added to the total number of AI Builder credits owned at your tenant level and will be lost.
+Some user licenses include seeded AI Builder credits. (eg: the Power Apps per user license includes 500 seeded AI Builder credits.) If **AI Builder capacity Per User add-on** isn't selected, the 500 AI Builder credits are still added to the total number of AI Builder credits owned at your tenant level and can be used.
+
+#### What happens to my data and models when my AI Builder trial expires?
+
+Data and models are kept within your environment and are not deleted. Data and models are deleted when the environment is deleted.
```
Comment (chplanty) [inline ai-builder-docs/administer-licensing.md:183]: Data and models are only deleted when ....

**Review by chplanty — APPROVED** [review]

File: ai-builder-docs/administer-licensing.md (hunk 1 above)
Comment (v-aangie) [reply ai-builder-docs/administer-licensing.md:171]: Should be one word.

**Comment by v-aangie** [comment]
@Antoine2F - Thanks for this PR. Going forward, please enter PRs in the private repo (not public). Changes in the public repo can go live without being reviewed. I've moved this to the private repo and it should go live later today. Closing this PR.

**Pull request closed without merging**